incubation_process.md

```diff
@@ -41,20 +43,24 @@ The barrier for entering the Sandbox stage is intended to be low, so that it's e
 
 ## Stage 2: Incubation
 
-The Incubation stage is for projects that have reached a level of maturity that indicates they are ready to be used by end users. Incubation projects meet the main criteria of the OpenRail Association which make sure that the project is governed openly. Open governance means that decisions are taken in a transparent way and there is a path to become committer and maintainer based on the merits of contributions.
+The Incubation stage is for projects that have reached a high level of quality and maturity that indicates they are ready to be used by end users. Incubation projects meet the main criteria of the OpenRail Association which make sure that the project is governed openly. Open governance means that decisions are taken in a transparent way and there is a path to become committer and maintainer based on the merits of contributions.
+
+The Incubation stage focuses on quality criteria the project can reach on its own and doesn't require a high degree of external adoption or contributions yet.
 
 The barrier for entering the Incubation stage is intended to be high, so there is a high level of quality for projects which have reached this stage.
 
 ### Criteria
 
-*Examples from CNCF incubation process, to be adopted*
-
-* Have a healthy number of committers. A committer is defined as someone with the commit bit; i.e., someone who can accept contributions to some or all of the project.
+* Have a healthy number of committers. A committer is defined as someone with the commit bit; i.e., someone who can accept contributions to some or all of the project. Committers can come from the same organization.
 * Demonstrate a substantial ongoing flow of commits and merged contributions.
-* Since these metrics can vary significantly depending on the type, scope and size of a project, the TOC has final judgement over the level of activity that is adequate to meet these criteria
-* A clear versioning scheme.
+* Since these metrics can vary significantly depending on the type, scope and size of a project, the TC has final judgement over the level of activity that is adequate to meet these criteria.
+* Have a clear release process which communicates to users what to expect from releases. This includes a clear versioning scheme and cryptographically signing release artifacts.
 * Specifications must have at least one public reference implementation.
+* Project has implemented an open governance process. It's documented how decisions are taken in the project and how people can become committers and maintainers.
 * Project explains the business roadmap.
+* Project is used in production. It's sufficient if this is only the organization developing the project.
+* Create clarity about licensing by being [REUSE](https://reuse.software) compliant.
```

Comment:
This includes the following aspects (as discussed in TC):

* Be REUSE compliant, the declaration with the toml file is a valid option
* Check REUSE compliance automatically (use standard GitHub action)
* Recommendation to add license headers to all files where technically possible